src/plots/polar/polar.js

```diff
@@ -1071,6 +1093,31 @@ function pathSectorClosed(r, sector) {
         (isFullCircle(sector) ? '' : 'L0,0Z');
 }
 
+// inspired by https://gist.github.com/buschtoens/4190516
+function pathAnnulus(r0, r1, _sector) {
+    var sector = _sector.slice();
+    var s0 = deg2rad(sector[0]);
+    var s1 = deg2rad(sector[1]);
+
+    // make a small cut on full sectors so that the
+    // inner region isn't filled
+    if(isFullCircle(sector)) s1 -= 1e-3;
```

Comment:
FWIW pies solved this by making [two arcs at each radius](https://github.com/plotly/plotly.js/blob/ef181daa3f18483af769cb776059b9709b17311b/src/traces/pie/plot.js#L216-L219) so the annulus really is complete.

*Probably* not going to cause any issues, it's just missing a fraction of a pixel, but I worry a little about numerical stability - it has to determine the center of the circle based on the very small vector from `p00` to `p01` and I'm not sure how much precision SVG uses.